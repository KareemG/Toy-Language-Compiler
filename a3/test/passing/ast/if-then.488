% If then statement

{
    if true then
        write "PASS"
}