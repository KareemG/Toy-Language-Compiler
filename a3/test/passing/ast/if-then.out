 { 
 if true then
 write "PASS"end
 }
